macast/utils.py

```diff
@@ -238,7 +240,28 @@ def set_start_at_login(launch):
                      'to delete login item "{}"'.format(app_name)])
             return res
         else:
-            return (1, 'Not support current platform.')
+            """Find the path of Macast.exe so as to create shortcut.
+            """
+            shell = client.Dispatch("WScript.Shell")
+            StartUp_Path = os.getenv('APPDATA') + r'\Microsoft\Windows\Start Menu\Programs\Startup'
```

Comment:
a better way to set startup_path (and all the variable names should be changed to lowercase to ensure consistency.)
```
startup_path = r'%APPDATA%\Microsoft\Windows\Start Menu\Programs\Startup'
```